app/renderer/lib/appium-client.js

```diff
@@ -211,11 +211,13 @@ export default class AppiumClient {
       if (_.toLower(platformName) === 'android' && _.toLower(automationName) === 'uiautomator2') {
         // returned Android height and width can both be affected by UiAutomator2 calculations
         // we stick with device dimensions, but swap them depending on detected orientation
+        // deviceScreenSize value fits portrait mode for phones, but landscape mode for tablets
         const [width, height] = deviceScreenSize.split('x');
-        if (windowSize.height > windowSize.width) { // portrait mode
+        if ((windowSize.height > windowSize.width && height > width) || // phone portrait mode
```

Comment:
should it be `>=` instead of `>` ?

Reply:
It's definitely an edgecase. If the dimensions are equal for both `windowSize` and `deviceScreenSize`, then either branch is fine. The only way problems could arise is if `width == height`, but `windowSize.height != windowSize.width`. I'm not sure how likely this is to actually happen.

Reply:
I'm also finding this condition a bit hard to read. Would it make more sense to reverse it?

Reply:
Created #1163 to adjust this